### ISSUE TITLE
FileNotFoundError reported for unsaved files

### ISSUE THREAD

**Issue by dmartin** [microsoft/vscode-flake8#97]
See example below:

https://user-images.githubusercontent.com/1657652/211078520-14bb7c79-81d7-499d-a936-4d6293838b7c.mov

Version info:

```
VSCode:

Version: 1.74.2
Commit: e8a3071ea4344d9d48ef8a4df2c097372b0c5161
Date: 2022-12-20T10:26:09.430Z (2 wks ago)
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 22.2.0
Sandboxed: No

Extension: v2022.6.0
```

**Comment by karthiknadig** [comment]
@dmartin please share the logs from Output > flake8.

**Comment by dmartin** [comment]
Sure, the relevant log is:

```
/usr/local/bin/python -m flake8 --format='%(row)d,%(col)d,%(code).1s,%(code)s:%(text)s' --toml-config=api/pyproject.toml Untitled-3
CWD Linter: /platform
untitled:Untitled-3 :
'0,1,E,E902:FileNotFoundError: [Errno 2] No such file or directory: 'Untitled-3''
```

(The `--toml-config` flag is due to my configuration:)

```
 "flake8.args": [
    "--toml-config=api/pyproject.toml"
  ],
  "flake8.importStrategy": "fromEnvironment",
```

This is inside a devcontainer, if that is possibly a contributing factor.

**Comment by karthiknadig** [comment]
The error itself is reported by `flake8`. All the extension is doing is showing what flake8 is giving us as a problem for that line. Please file this on https://github.com/PyCQA/flake8/issues .

**Comment by karthiknadig** [comment]
Ah! sorry missed that. Yes, sorry about that. 

This should be easy enough of a check to add. I forgot that for `flake8` we were not able to use the `stdin` to pass the unsaved content to get the result and we had to rely on using the file on disk to get the issues. For other linters we were able to use `stdin` to pass the content, so unsaved buffer did not matter.
